lib/companies/schema/job.ex

```diff
@@ -7,6 +7,7 @@ defmodule Companies.Schema.Job do
   schema "jobs" do
     field :title, :string
     field :url, :string
+    field :remote, :boolean
```

Comment:
Nit: What about when a job is not just remote or not? What if it is "remote for the right candidate"?

Reply:
That's a good question @pdgonzalez872! I think a note could go in the job description. On other job sites I've seen those jobs listed as on-site with a note about remote for the right person.

Thoughts @maartenvanvliet? @gemantzu?

Reply:
don't think it needs to be covered right now, as said a note could go in a description.

Reply:
@doomspork I think you are right, we can not cover all cases like that, but as an afterthought we could turn it into a dropdown selection ("remote", "on-site", "remote for the right person") as a new Schema (JobLocation), and we could keep adding to it as needed. 

But I believe this is not actually needed, simply adding a remote field should suffice for 99% of the use cases.